CHANGELOG.md

```diff
@@ -3,7 +3,16 @@
 All notable, unreleased changes to this project will be documented in this file. For the released changes, please visit the [Releases](https://github.com/mirumee/saleor/releases) page.
 
 # 3.8.0 [Unreleased]
-
+### Highlights
+- Improve support for handling transactions - #10350 by @korycins
```

Comment:
Here also should be an empty line after `Highlights`

Reply:
Solved in: https://github.com/saleor/saleor/pull/10687